### PR TITLE
WIP: rm deprecated pages

### PR DESCRIPTION
**Pull request by mekarpeles**
We should add to this:
- [ ] rm `/account/preferences` (the view and the html template) as they are deprecated

### CODE REVIEW

**Comment by hornc** [comment]
thanks @mekarpeles , I have not yet reviewed, but this will help me with my planned clean-up of i18n strings in #801 :+1:

**Comment by mekarpeles** [comment]
We should add to this `/account/preferences` (the view and the html template) as they are deprecated

File: openlibrary/plugins/openlibrary/borrow_home.py [hunk 1]
```diff
@@ -75,10 +69,7 @@ class read(delegate.page):
     path = "/read"
 
     def GET(self):
-        rand = random.randint(0, 9999)
-        sort = "random_%d desc" % rand
-        subject = get_readable_books(web.ctx.site, details=True, limit=12, sort=sort)
-        return render_template("borrow/read", subject, stats=LoanStats(), rand=rand)
+        web.seeother('/subjects/accessible_book#ebooks=true')
```
Comment (hornc) [inline openlibrary/plugins/openlibrary/borrow_home.py]: @mekarpeles class `borrow_about(delegate.page):` still exists below but its template has been deleted, /borrow/about errors.

**Comment by hornc** [comment]
Also removing borrow/navigation templates has broken the admin only /libraries/dashboard page -- is that to be removed too? I don't know that has any function currently. I'm happy to remove that in a subsequent PR as it is admin only (but removal will help with the i18n string tidy)

**Comment by mekarpeles** [comment]
We should keep the library endpoints, I have to go back and add borrow/navigation back in

**Comment by mekarpeles** [comment]
I'm tempted to close this and re-apply with the parts we know can be removed @hornc, @cdrini 

@cdrini is this something you'd be interested in tackling? Goal is to eliminate a few html templates which are deprecated

**Comment by cdrini** [comment]
Interested, yes; although not sure when I'll have the bandwidth :P Maybe close this, and open a new issue for removing deprecated templates (which references the PR)? We can then have more focussed PRs for removing specific groups of pages.

**Pull request closed without merging**